pillar.example

```diff
@@ -118,26 +118,28 @@ nginx:
       # see also http://nginx.org/en/docs/example.html
       #-- - - - - -- - - -- - - - - -- - - -- - - - -- - - - - - -- - - - - - -- - - - - -- - - - - -- - - #
       config:
-        include 'snippets/letsencrypt.conf'
+        include: 'snippets/letsencrypt.conf'
         source_path: salt://path_to_nginx_conf_file/nginx.conf # IMPORTANT: This option is mutually exclusive with the rest of the
                                                           # options; if it is found other options (worker_processes: 4 and so
                                                           # on) are not processed and just upload the file from source
         worker_processes: 4
         load_module: modules/ngx_http_lua_module.so  # pass as very first in configuration; otherwise nginx will fail to start
         #pid: /var/run/nginx.pid                     # Directory location must exist (i.e. it's  /run/nginx.pid on EL7)
         events:
-          worker_connections: 768
+          worker_connections: 1024
         http:
           sendfile: 'on'
           include:
             #### Note: Syntax issues in these files generate nginx [emerg] errors on startup.  ####
             - /etc/nginx/mime.types
 
-        ### module ngx_http_log_module example
-        log_format: |-
-          main '...';
-            access_log /var/log/nginx/access_log main
-        access_log: []       #suppress default access_log option from being added
+          ### module ngx_http_log_module example
+          log_format: |-
+            main '$remote_addr - $remote_user [$time_local] "$request" '
+                                '$status $body_bytes_sent "$http_referer" '
+                                '"$http_user_agent" "$http_x_forwarded_for"';
+              access_log /var/log/nginx/access.log main
```

Comment:
@garrettw Nearly there!  As [mentioned above](https://github.com/saltstack-formulas/nginx-formula/pull/217#discussion_r274222091), this line is erroneous and needs to be removed.

Not as important but from the same comment, the upstream example has `"$request"` and `$status` the other way around.  Would you mind switching these over?

Reply:
How is line 141 erroneous? I thought it was needed based on #122.

Reply:
@garrettw Three reasons:

1. Where it is currently positioned (indented), it gets included in the `log_format`, as shown in the previous comment.
1. Even if indented correctly, it would need to be supplied with the missing colon.
1. There's already an entry for `access_log` on the line below (line 142).